libs/utils/utils.js

```diff
@@ -730,28 +730,34 @@ async function decorateIcons(area, config) {
 async function decoratePlaceholders(area, config) {
   const el = area.querySelector('main') || area;
   const regex = /{{(.*?)}}|%7B%7B(.*?)%7D%7D/g;
-  const walker = document.createTreeWalker(
-    el,
-    NodeFilter.SHOW_TEXT,
-    {
-      acceptNode(node) {
-        const a = regex.test(node.nodeValue) ? NodeFilter.FILTER_ACCEPT : NodeFilter.FILTER_REJECT;
-        regex.lastIndex = 0;
-        return a;
-      },
-    },
-  );
+  const walker = document.createTreeWalker(el, NodeFilter.SHOW_ALL, null, false);
```

Comment:
I'm a bit confused by the last 2 arguments. According to https://developer.mozilla.org/en-US/docs/Web/API/Document/createTreeWalker, the filter is optional, so it may be omitted, while `false` is a 4th argument that is not mentioned in their documentation. What is its purpose?

Reply:
I wondered the same. It's `entityReferenceExpansion` https://www.w3.org/TR/WD-DOM-Level-2/traversal.html

Reply:
From what I've seen, the proposal didn't make it into the final draft: https://dom.spec.whatwg.org/#dom-document-createtreewalker

hence I'd say we can omit the 4th argument.

Reply:
And does it bring any advantages? From a quick search I could only find that it was useful for IE11 🤷

Reply:
Given the living standard, it seems that it isn't part of the specification at all anymore `createTreeWalker(root, whatToShow, filter)`